bazel/emscripten_toolchain/toolchain.bzl

```diff
@@ -1060,6 +1061,10 @@ def _impl(ctx):
                     key = "EM_CONFIG_PATH",
                     value = ctx.file.em_config.path,
                 ),
+                env_entry(
+                    key = "NODE_JS_PATH",
+                    value = nodejs_path,
```

Comment:
If this is is an absolute path (or can be) then you can just pass `EM_NODE_JS` and completely remove the line from the config file.

Reply:
I looked a bit more into this, but couldn't find a way to do this much better. When this code here is executed, then the absolute path is not known yet, as we aren't yet executing actions, which run in sandboxes. I looked for some variable that will get substituted later, but couldn't find any.

There is env.sh, which is called before we end up in the config and it could do the conversion to an absolute path, but that doesn't really save any work. If we do it in env.sh, it does not work on Windows, which has its own env.bat, so the conversion forks into two places.